package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apple/app-store-server-library",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
If you are going to bump the version, please also bump the version in the user agent string as well